### PR TITLE
Fixes to use dygraphs 1.1

### PR DESCRIPTION
**Pull request by chrisirhc**
- Add default `xAxisLabelFormatter` when no xAxisLabelFormatter is specified
- Default to display MMM YY for monthly x axis labels
- Allow using legend `show="follow"`
- Move axisLabelWidth option to axes options

screen shot 2015-10-10 at 3 04 51 pm

### CODE REVIEW

**Comment by chrisirhc** [comment]
Just saw that #84 has some overlap with this with regards to deprecated options. Those changes aren't important to this PR so I'm happy to remove / rebase them for this PR.

**Comment by jjallaire** [comment]
Unfortunately the change to 60 pixels is not enough to prevent a huge decrease in the number of labels on the y-axis in some cases (e.g. try the example code here with the change: http://rstudio.github.io/dygraphs/gallery-axis-options.html).

File: R/options.R [hunk 1]
```diff
@@ -206,8 +200,6 @@ dyOptions <- function(dygraph,
     options$colors <- as.list(colors)
   options$colorValue <- colorValue
   options$colorSaturation <- colorSaturation
-  options$drawXAxis <- drawXAxis
-  options$drawYAxis <- drawYAxis
```
Comment (jjallaire) [inline R/options.R]: Here we're breaking existing code that passes those values. The values need to be set to the correct spot within the individual axis options (or you could drop this code in favor of the other PR).

Comment (chrisirhc) [inline R/options.R]: Yea, I'll drop these changes instead since they're not required for this PR.

**Comment by chrisirhc** [comment]
Ah, okay, so it sounds like you'd like to retain the old axis time formatting behavior. I've modified my PR to set the `xAxisLabelFormatter`. The example you mentioned looks like:

screen shot 2015-10-10 at 3 04 51 pm

- Removed changes to axes settings

**Comment by jjallaire** [comment]
Subsumed by https://github.com/rstudio/dygraphs/issues/89

**Pull request closed without merging**